dashboard/test/controllers/pd/professional_learning_landing_controller_test.rb

```diff
@@ -168,23 +168,6 @@ def prepare_scenario
 
   test_redirect_to_sign_in_for :index
 
-  test 'teachers without enrollments are redirected' do
```

Comment:
I removed these tests since they seemed irrelevant now that we aren't redirecting this page.